src/server/helpers/token.js

```diff
@@ -8,6 +8,10 @@ dotenv.config();
 const { SECRET } = process.env;
 const tokenLife = process.env.TOKEN_LIFE || '30m';
 
+export const verify = async token => jwt.verify(token, SECRET);
+
+export const decode = async token => jwt.decode(token);
```

Comment:
I would suggest using better name for functions

Reply:
it just method overloading base jwt lib. I used this methods like Token.verify()